packages/lms/src/routes/+layout.svelte

```diff
@@ -1,4 +1,81 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
+  import '@skeletonlabs/skeleton/themes/theme-skeleton.css';
+  import '@skeletonlabs/skeleton/styles/skeleton.css';
+  import '../lib/styles/styles.scss';
+  import { Molly } from '@mollify/molly';
+  import '../app.postcss';
+  import { AppShell, Drawer, Toast } from '@skeletonlabs/skeleton';
+  import Main from '$lib/components/content/Main.svelte';
+  import Footer from '$lib/components/footer/Footer.svelte';
+  // import Reader from '$lib/components/header/Reader.svelte';
+  import IconNav from '$lib/components/navigation/IconNav.svelte';
+  import Header from '$lib/components/header/Header.svelte';
+  import 'prismjs/themes/prism-tomorrow.css';
+  import type { LayoutData } from './$types';
+  import EntityNav from '$lib/components/navigation/EntityNav.svelte';
+  import { page } from '$app/stores';
+
+  export let data: LayoutData;
+
+  const scrollIntoView = (node: HTMLElement) => {
+    node.scrollIntoView();
+  };
+
+  let isMobile = false;
+
+  // Function to check if the screen width is below a certain breakpoint (e.g., 768px)
+  const checkIsMobile = () => {
+    isMobile = window.innerWidth < 768;
+  };
+
+  // Initial check on component mount
+  onMount(() => {
+    checkIsMobile();
+    // Listen for window resize events to update the isMobile variable
+    window.addEventListener('resize', checkIsMobile);
```

Comment:
Is it perhaps better to utilize the `<svelte:window>` utility [Svelte provides](https://learn.svelte.dev/tutorial/svelte-window)?
Maybe it's better to run it when the component is mounted, but this should be tested in my opinion.

Reply:
A reactive prop would also work

Reply:
Because this is not reactive, changing from one size to another in a browser disables the mobile nav menu.